### ISSUE TITLE
ParseError in PHP 7 results in uncaught TypeError

### ISSUE THREAD

**Issue by kiler129** [symfony/symfony#15245]
I found that sometime Symfony prints exception as below:

```
Fatal error: Uncaught TypeError: Argument 1 passed to Symfony\Component\Debug\Exception\FatalBaseException::__construct() must be an instance of BaseException, instance of ParseError given, called in /directory/vendor/symfony/symfony/src/Symfony/Component/Debug/ErrorHandler.php on line 442 and defined in /directory/vendor/symfony/symfony/src/Symfony/Component/Debug/Exception/FatalBaseException.php:21
Stack trace:
#0 /directory/vendor/symfony/symfony/src/Symfony/Component/Debug/ErrorHandler.php(442): Symfony\Component\Debug\Exception\FatalBaseException->__construct(Object(ParseError))
#1 [internal function]: Symfony\Component\Debug\ErrorHandler->handleException(Object(ParseError))
#2 {main}
  thrown in /directory/vendor/symfony/symfony/src/Symfony/Component/Debug/Exception/FatalBaseException.php on line 21
```

Problem was of course caused by typo in my code, but it should be catched ;) Correct me if I'm wrong but I think there's a problem because PHP 7 introduced `Throwable` along with `BaseError`.
Does Symfony 2.7 supports PHP 7?

**Comment by jvasseur** [comment]
I think this was fixed in #14959.

**Comment by kiler129** [comment]
You're right. Strange, should I be getting latest 2.7 release with `"symfony/symfony": "~2.7@beta"` entry? With `2.7-dev` I'll get `dev-master`.

**Comment by jvasseur** [comment]
It's not in the last released 2.7 version yet, you should be able to get it with a constraint like "2.7.*@dev".
